helm/reana/templates/serviceaccount.yaml

```diff
@@ -1,5 +1,19 @@
+---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "reana.prefixed_svaccount_name" . }}
-  namespace: {{ .Values.serviceAccount.namespace }}
+  name: {{ include "reana.prefixed_infrastructure_svaccount_name" . }}
```

Comment:
The service accounts ends up being `reana-reana-infrastructure` by default, is this expected?